src/components/ProjectPreview.jsx

```diff
@@ -1,14 +1,55 @@
+import {faArchive} from '@fortawesome/free-solid-svg-icons';
+import {FontAwesomeIcon} from '@fortawesome/react-fontawesome';
+import classnames from 'classnames';
 import React from 'react';
 import PropTypes from 'prop-types';
 import moment from 'moment';
+import {t} from 'i18next';
 
 const MAX_LENGTH = 50;
 
-export default function ProjectPreview({preview, project}) {
+export default function ProjectPreview({
+  isCurrentProject,
+  preview,
+  project,
+  onProjectArchived,
+}) {
   return (
     <div>
-      <div className="project-preview__label">
-        {preview.slice(0, MAX_LENGTH)}
+      <div className="project-preview">
+        <div
+          className={classnames('project-preview__label', {
+            'project-preview__label_archived': project.isArchived,
+          })}
+        >
+          {preview.slice(0, MAX_LENGTH)}
+        </div>
+        {
+          (function showArchived() {
+            if (isCurrentProject) {
+              return null;
+            } else if (project.isArchived) {
```

Comment:
I would expect the linter to catch this, but no need for an `else` when the `if` branch returns.